vm/environment.h

```diff
@@ -176,7 +176,7 @@ class Environment : public ISourcePawnEnvironment
   bool profiling_enabled_;
 
   ke::AutoPtr<CodeAllocator> code_alloc_;
-#if defined(SP_HAS_JIT)
+#if defined(SP_HAS_JIT) || defined(KE_ARCH_X64)
```

Comment:
I think it's okay to just require that all archs support CodeStubs. Then we can take out some of these ifdefs.